config/locales/doorkeeper.en.yml

```
@@ -0,0 +1,151 @@
+en:
+  activerecord:
+    attributes:
+      doorkeeper/application:
+        name: 'Name'
```

Should we align on the format of `yml` files, either keep quotes or remove unnecessary quotes

```suggestion
        name: "Name"
```

or

```suggestion
        name: Name
```

---

Hmm, I think the double quotes are better since the other `yml` files have double quotes too 🤔  Great suggestion 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/31/commits/a3ec449edaed5d16a742c05e2ffbe5d02f27275b